### ISSUE TITLE
docker container doesn't build

### ISSUE THREAD

**Issue by fferner** [infiniband-radar/infiniband-radar-daemon#8]
Hi,

after initial tests last year, we are now trying to roll this out further. As part of this I noticed that the docker container in option 1 of the readme no longer builds. Presumably due to changes in the alpine linux repositories. The dependency for infiniband-diags (libosmcomp.so.3) can no longer be resolved.

Trying to build is returns this error:

> sudo docker build -t infiniband_radar_daemon .
Sending build context to Docker daemon  1.847MB
Step 1/14 : FROM alpine:latest as builder
 ---> cc0abc535e36
Step 2/14 : RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/testing/ >> /etc/apk/repositories &&     apk add --no-cache infiniband-diags-dev curl-dev g++ make cmake
 ---> Running in 5f4c4ff5f539
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  so:libosmcomp.so.3 (missing):
    required by: infiniband-diags-1.6.6-r2[so:libosmcomp.so.3]
The command '/bin/sh -c echo http://dl-cdn.alpinelinux.org/alpine/edge/testing/ >> /etc/apk/repositories &&     apk add --no-cache infiniband-diags-dev curl-dev g++ make cmake' returned a non-zero code: 3

I tried finding an older aline repo which might resolve this but have not succeeded in this.

**Comment by carstenpatzke** [comment]
I'm incredible sorry for leaving this issue untouched for so long.
Thank you for the detailed error report.

I had to use the edge (experimental) repository of alpine, in order to get all the necessary InfiniBand libraries, but it seems that they changed the libosmcomp major version, which broke the building process.

The new version runs on debian:buster-slim.

I hope the issue was fixed in the latest commit (f64b42347f88a4024c1a7d9d82ccd72577a5d94f).
